webapp/apps/dynamic/models.py

```diff
@@ -87,6 +87,25 @@ class DynamicBehaviorSaveInputs(models.Model):
     micro_sim = models.ForeignKey(OutputUrl, blank=True, null=True,
                                   on_delete=models.SET_NULL)
 
+    def get_tax_result(self):
+        """
+        If taxcalc version is greater than or equal to 0.13.0, return table
+        If taxcalc version is less than 0.13.0, then rename keys to new names
+        and then return table
+        """
+        outputurl = OutputUrl.objects.get(unique_inputs__pk=self.pk)
+        taxcalc_vers = outputurl.taxcalc_vers
+        taxcalc_vers = taxcalc_vers.split('.')
```

Comment:
Can simplify to

```
taxcalc_vers = outputurl.taxcalc_vers.split('.')
```

But probably the best solution would be to use some versioning comparison standard library, i.e. `distutils.version.StrictVersion` or `distutils.version.LooseVersion` so that you don't have to mess around with this version comparison stuff.